rssalertbot/alerts.py

```diff
@@ -20,16 +33,26 @@ def alert_email(feed, cfg, entry):
         cfg (dict):              output config
         entry (dict):            the feed entry
     """
-    log.debug(f"Alerting email: {entry.title}")
+    logger = logging.LoggerAdapter(log, extra = {
```

Comment:
Maybe this loggingAdapter change can be pulled out into a small helper method in `util`? Since it's called all over the place in seemingly the exact same way.